saleor/graphql/account/mutations.py

```diff
@@ -297,6 +297,21 @@ def perform_mutation(cls, root, info, **data):
         return super().perform_mutation(root, info, **data)
 
 
+class AccountDelete(BaseMutation):
```

Comment:
This name can be confusing since this mutation doesn't delete the account yet. I would rename it to `AccountDeleteEmail`.

For every operation that requires an email confirmation, we'll need two mutations: one to send an email and the other actually perform the action once users click the link. But I think we'll do the refactor of that operation in a separate PR.

Reply:
`AccountDeleteEmail` is confusing me as well. What about `AccountRequestDelete`/`AccountRequestDeletion`?

Reply:
Right, `AccountRequestDeletion` sound good.